src/js/helpers/searchHelper.js

```diff
@@ -258,6 +258,23 @@ export const fetchAwardTransaction = (params) => {
     };
 };
 
+// Fetch IDV Award Federal Account Funding Data
```

Comment:
Could you move this to `helpers/idvHelper.js`? This file should be for the search pages, and when other pages access a shared endpoint. New endpoints specific to IDVs should go in a separate file.